config/routes.rb

```diff
@@ -1,9 +1,6 @@
 Rails.application.routes.draw do
   root 'items#home'
 
-  get '/home', to: 'items#home'
-  post '/items', to: 'items#create'
-  patch '/items', to: 'items#update'
-
-  post '/categories', to: 'categories#create'
+  resources :items, only: [:create]
```

Comment:
Thanks, good catch. There is actually no update action in the controller so I thought this was unintentional - but I added back.